deblur/workflow.py

```diff
@@ -41,9 +47,95 @@ def dereplicate_seqs(seqs_fp,
         log_name=log_name)
 
 
-def remove_artifacts_seqs(seqs_fp):
-    """Step 3: remove artifacts from FASTA file"""
-    pass
+def remove_artifacts_seqs(seqs_fp,
+                          ref_fp,
+                          output_fp,
+                          ref_db_fp=None,
+                          negate=False,
+                          threads=1):
+    """Step 3: remove artifacts from FASTA file using
+       SortMeRNA
+
+    Parameters
+    ----------
+    seqs_fp: string
+        file path to FASTA input sequence file
+    ref_fp: tuple
```

Comment:
Ugh... I think we should discourage that behavior, since building the SortMeRNA database is expensive, we potentially want to do it only once....
Comment not blocking though....